src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.js

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
```

Comment:
my recommendations to useField

Reply:
imo Field is more readable + there are conditional fields here